csharp/src/Google.Protobuf/Collections/MapField.cs

```diff
@@ -53,6 +53,11 @@ namespace Google.Protobuf.Collections
     /// For string keys, the equality comparison is provided by <see cref="StringComparer.Ordinal" />.
     /// </para>
     /// <para>
+    /// Null values are not permitted in the map, either for wrapper types or regular messages.
+    /// If a map is deserialized from a data stream and the value is missing from an entry, and the entry
```

Comment:
MapWithOnlyKey_PrimitiveValue tests that such map is parseable and the value will be set to the default message.